bin/sdk-cli.js

```diff
@@ -71,25 +71,18 @@ const build = ( target, argv ) => {
 yargs
 	.scriptName( scriptName )
 	.usage( `Usage: $0 <command> ${ delimit }[options]` )
-	.example( `$0 gutenberg ${ delimit }--editor-script=hello-dolly.js` )
+	.example( `$0 gutenberg ${ delimit }hello-dolly` )
```

Comment:
we probably need to add more here, a real path, otherwise this example seems somewhat unhelpful

```js
`$0 gutenberg ${ delimit }gutenberg/extensions/hello-doly`
```